packages/code/src/Code.tsx

```diff
@@ -70,6 +74,14 @@ const codeWrapperStyleWithWindowChrome = css`
   border-left: 0;
 `;
 
+const copyStyle = css`
+  min-width: 38px;
```

Comment:
How come we don't just use `width` here?

Reply:
I found that when I just used `width`, as more text was added and the element widened, the sidebar shrank from 38px to like 29px. If I set `min-width` instead, that shrinkage didn't happen.

Reply:
Ahh, its parent is `display: flex`. You should be able to use `width` in combination with `flex-shrink: 0`.